src/XrdHttp/XrdHttpReq.cc

```diff
@@ -2692,8 +2692,12 @@ int XrdHttpReq::PostProcessHTTPReq(bool final_) {
     {
 
       if (xrdresp != kXR_ok) {
-        prot->SendSimpleResp(httpStatusCode, NULL, NULL,
-                             httpStatusText.c_str(), httpStatusText.length(), false);
+        if (xrderrcode == kXR_ItExists) {
+          prot->SendSimpleResp(405, NULL, NULL, (char *) "Method is not allowed", 0, false);
```

Comment:
Hmmm, can you also add "; resource already exists." to the text so that at least the person getting the message knows why it cannot be done?

Reply:
Sure, I've updated the return text.